### ISSUE TITLE
support for simple 1d bifurcations

### ISSUE THREAD

**Issue by harsh306** [nschloe/pacopy#13]
Saddle: u + x^2
Transcritical: ux - x^2
Pitchfork: ux - x^3

**Comment by nschloe** [comment]
What do you mean by "support"?

**Comment by harsh306** [comment]
Currently, in this repo we have natural and pseudo arc-length continuation methods. It will be great if more types bifurcation points could be detected. Also above equations can be provided as a few more example bifurcation points. 

I am sorry, I am still new in this area, so my explanation could be naive.

**Comment by nschloe** [comment]
Yeah, I mean, it'd be great to have bifurcation point detection + branch switching. That's a challenging topic, but I've also been wanting to look into this for a while.
